DevProxy/Program.cs

```diff
@@ -32,12 +32,14 @@ static WebApplication BuildApplication(string[] args, DevProxyConfigOptions opti
 
     return app;
 }
-_ = Announcement.ShowAsync();

```

Comment:
Originally, we haven't registered Announcement as a service so that we can run it earlier before the whole service container is instrumented. Since it's executed just once in the application, I suggest we keep it that way.

Reply:
reverted